web/src/components/Registration.js

```diff
@@ -0,0 +1,267 @@
+    const reducer = (acc, item) => ({[item]: acc});
+    const changes = parents.length === 0 ? {[field]: {$set: value}} : parents.reduce(reducer, {[field]: {$set: value}});
+
+    // Create an immutable copy of the data first to avoid changes by reference.
+    const formData = update(this.state.formData, changes);
+    this.setState({formData: formData})
+  }
+
+  render() {
+    const summaryFormData = JSON.stringify(this.state.formData, null, "  ");
+
+    return (
+      <Form noValidate validated={this.state.validated} onSubmit={this.handleSubmit}>
+        <Tab.Container id="registration-form" activeKey={this.state.tabKey} onSelect={(k) => this.setState({tabKey: k})}>
+          <Row className="pt-3 pb-5">
+            <Col sm={3}>
+              <Nav variant="pills" className="flex-column">
+                <Nav.Item>
+                  <Nav.Link eventKey="introduction">Introduction</Nav.Link>
+                  <Nav.Link eventKey="entity-details">Entity Details</Nav.Link>
+                  <Nav.Link eventKey="trisa-implementation">TRISA Implementation</Nav.Link>
+                  <Nav.Link eventKey="contacts">Contacts</Nav.Link>
+                  <Nav.Link eventKey="trixo">TRIXO Questionnaire</Nav.Link>
+                  <Nav.Link eventKey="summary">Summary</Nav.Link>
+                </Nav.Item>
+              </Nav>
+            </Col>
+            <Col sm={9}>
+              <Tab.Content>
+                <Tab.Pane eventKey="introduction">
+                  <fieldset>
+                    <legend>Introduction</legend>
+                    <p>
+                      Thank you for your interest in the TRISA network for Travel Rule Compliance.
+                      This multi-part form is the first step in the registration and certificate
+                      issuance process. The information you provide will be used to verify the
+                      legal entity that you represent and, where appropriate, will be available to
+                      verified TRISA members to facilitate compliance-decisions.
+                    </p>
+                    <p>
+                      To assist in completing the registration form, which is somewhat lengthy, the
+                      form is broken into multiple sections, with information stored in your <em>local
+                      browser storage</em> so that you can come back and complete the process. <strong>No
+                      information is sent until you submit the form in the summary section</strong>.
+                    </p>
+                    <p className="text-danger">
+                      The registration form is currently in its beta implementation. You will not be
+                      able to submit the form now, but can download the form to submit it via the CLI
+                      registration interface. The registration form will be completed when the TRISA
+                      v1beta1 protocol is released.
```

Comment:
Suggest updating the copy to read:

This registration form is currently in its beta implementation. You will not be able to submit the form now, but can download it and submit via the CLI registration interface. This registration form will become live when the TRISA v1beta1 protocol is released.